specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-11-01/managedClusters.json

```diff
@@ -1945,6 +1945,22 @@
       },
       "description": "Identity for the managed cluster."
     },
+    "UserAssignedIdentity": {
```

Comment:
@norshtein thoughts on making this UserAssignedIdentityProfile so that we follow the current `Profile` pattern?